docker-compose.yml

```
@@ -29,7 +27,7 @@ services:
     expose:
       - "8000"
     environment:
-      - DATABASE_URL=postgres://postgres:password@postgres:5432/mydb
+      - DATABASE_URL=postgres://postgres:flashqc:1234@localhost/flashqc
```

The 502 error you see at `http://localhost:8080/api/html-tags` when running `mise dserve` on this PR is caused by Rocket defaulting to launching at address 127.0.0.1, which means that it can only receive connections from inside its _container_ (which is not very helpful 🙃).

The fix is to set the address to be 0.0.0.0, so that other containers can talk to it.

There are several ways to achieve this. One of them is to specify something like

```suggestion
      - DATABASE_URL=postgres://postgres:flashqc:1234@localhost/flashqc
      - ROCKET_ADDRESS=0.0.0.0
      - ROCKET_PORT=8000
```

here.